Anlab.Mvc/Controllers/AdminController.cs

```diff
@@ -170,7 +170,13 @@ public IActionResult ListUsersOrders(string id)
             {
                 var orderToUpdate = await _dbContext.Orders.SingleAsync(a => a.Id == model.OrderId.Value);
 
-                await _orderService.PopulateOrder(model, orderToUpdate);
+                await _orderService.PopulateOrder(model, orderToUpdate, true);
```

Comment:
@jSylvestre don't love the mystery bool isAdmin, maybe it's better to just use User.IsInRole or do an overload PopulateLabOrder().